packages/vue-components/src/Overlay.vue

```diff
@@ -60,35 +47,21 @@ export default {
     };
   },
   methods: {
-    toggleNavMenu() {
-      if (!this.show) {
+    toggleNavMenu(ev) {
+      if (ev.target.tagName.toLowerCase() === 'a' || this.show) {
```

Comment:
```suggestion
      if (ev.target.tagName.toLowerCase() === 'a' && this.show) {
```
I think this should be `&&` so that the site nav title does not close the overlay when it is clicked?

[](https://user-images.githubusercontent.com/47470981/116780910-05044a80-aab2-11eb-9079-9c000c31d89a.mp4)

Reply:
Fixed! Still `||`, but added some extra checks, `&&` would disable the closing button.